libraries/AP_HAL_ChibiOS/hwdef/CUAV-7-Nano/hwdef.dat

```diff
@@ -0,0 +1,305 @@
+# hw definition file for processing by chibios_hwdef.py
+# for CUAV-7-Nano board
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# board ID for firmware load
+APJ_BOARD_ID 7000
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# to be compatible with the px4 bootloader we need
+# to use a different RAM_MAP
+env USE_ALT_RAM_MAP 1
+
+# flash size
+FLASH_SIZE_KB 2048
+
+# with 2M flash we can afford to optimize for speed
+env OPTIMIZE -O2
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART3 OTG2
+
+# default to all pins low to avoid ESD issues
+DEFAULTGPIO OUTPUT LOW PULLDOWN
```

Comment:
oh and for this to do anything, it must be before the MCU line

Reply:
Okay, I'll modify it, thanks.